content/docs/higher-order-components.md

```diff
@@ -156,7 +156,7 @@ function withSubscription(WrappedComponent, selectData) {
 
     render() {
       // ... and renders the wrapped component with the fresh data!
-      // Notice that we pass through any additional props
+      // Notice that we pass through an additional props
```

Comment:
there are mutliple props. it's like "we pass through any additional content" or "any additonal stuff". not a singular "props". Hope this makes sense

Reply:
It makes more sense now. 
Thanks for the reply @eps1lon